stable/python/kubeless.py

```diff
@@ -59,6 +73,7 @@ def handler():
         data = req.json
     event = {
         'data': data,
+        'files': req.files, # allow file uploads
```

Comment:
For consistency with the rest of runtimes we should not modify the `event` structure (which is documented [here](https://kubeless.io/docs/kubeless-functions/). If you want to access the `files` helper (which is only present in python) you can still access it trough `event.extensions.request.files`.